sound/hda/intel-dsp-config.c

```diff
@@ -472,6 +472,14 @@ static const struct config_entry config_table[] = {
 	},
```

Comment:
I note @plbossart  has used "ASoC" prefix in some commits (like a75bdee43cb1), but I don't think this is correct. This should be:

ALSA: hda: intel-dsp-config: add MTL PCI id

Reply:
> I note @plbossart has used "ASoC" prefix in some commits (like [a75bdee](https://github.com/thesofproject/linux/commit/a75bdee43cb1693e7733123af2a7117e33558717)), but I don't think this is correct. This should be:
> 
> ALSA: hda: intel-dsp-config: add MTL PCI id

ack